components/renku_data_services/data_connectors/api.spec.yaml

```diff
@@ -165,7 +165,69 @@ paths:
           $ref: "#/components/responses/Error"
       tags:
         - data connectors
-
+  /data_connectors/{data_connector_id}/project_links:
+    parameters:
+      - in: path
+        name: data_connector_id
+        required: true
+        schema:
+          $ref: "#/components/schemas/Ulid"
+        description: the ID of the data connector
+    get:
+      summary: Get all links from a given data connector to projects
+      responses:
+        "200":
+          description: List of data connector to project links
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/DataConnectorToProjectLinksList"
+        default:
+          $ref: "#/components/responses/Error"
+      tags:
+        - data connectors
```

Comment:
We use underscore to separate words in tags although not needed by OpenApi specs and spaces are allowed. I suggest we use the same convention here. Up to you to decide.

Reply:
Done in 968df0f89befff1c2e5564770fdccd0638479432.